core/src/main/java/io/fabric8/maven/core/access/ClusterAccess.java

```diff
@@ -72,6 +73,18 @@ public String getNamespace() {
         return namespace;
     }
 
+    /**
+     * Returns true if this cluster is a traditional OpenShift cluster with the <code>/oapi</code> REST API
+     * or supports the new <code>/apis/image.openshift.io</code> API Group
+     */
```

Comment:
the method `openShiftClient.supportsOpenShiftAPIGroup(OpenShiftAPIGroups.IMAGE)` returns true if the old /oapi REST API is supported or the new API Group